recipes/clark/1.2.2-b/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#strictly use anaconda build environment
+CC=${PREFIX}/bin/gcc
```

Comment:
I don't know, probably not. I left it in because it was in the previous version of the recipe, but I can try it without.